CONTRIBUTING.md

```diff
@@ -498,6 +498,16 @@ functionality should be added, each one will need their own super-set
 interfaces and will duplicate the pattern. For this reason, the simple targeted
 interface that defines the specific functionality should be preferred.
 
+### Shared code
```

Comment:
How do we plan to address the internal `otel` logging functionality?

Reply:
I will take a look tomorrow. Can you send a hyperlink with an example code that you are mentioning?

Reply:
https://github.com/open-telemetry/opentelemetry-go/blob/3df561e6446d5990db4aed37ab9c0a7a1ceadfce/internal/global/internal_logging.go#L61